Mathlib/CategoryTheory/Limits/Shapes/Terminal.lean

```diff
@@ -602,9 +602,9 @@ def limitOfDiagramInitial {X : J} (tX : IsInitial X) (F : J ⥤ C) :
     IsLimit (coneOfDiagramInitial tX F) where
   lift s := s.π.app X
   uniq s m w := by
-    conv_lhs => dsimp
+    show m = s.π.app X
```

Comment:
This is curious, does this mean the dsimp did do something?

Reply:
Actually, I think that `dsimp` was doing something, but when I was looking at it I misdiagnosed: currently, the linter performs poorly inside `conv`.